ztp/source-crs/ImageRegistryPvc.yaml

```diff
@@ -0,0 +1,15 @@
+kind: PersistentVolumeClaim
```

Comment:
We already have StoragePVC.yaml in the source-crs. Can the user simply use that source-cr (with name/namespace override) to avoid duplication?

Reply:
Removed the file and using existing source cr for PVC